templates/ui-frameworks/vanilla/example/workdir/happ.yaml.hbs

```diff
@@ -12,7 +12,6 @@ roles:
       modifiers:
         network_seed: ~
         properties: ~
-        origin_time: ~
```

Comment:
heads up `quantum_time` can be deleted here too

Reply:
Removed